src/main/java/hunternif/mc/atlas/SettingsConfig.java

```diff
@@ -70,6 +70,8 @@
         public boolean doScanRavines = true;
         @Config.Comment({"If true, map render time will be output."})
         public boolean debugRender = false;
+        @Config.Comment({"If true, to get more detailed data about visited marker "})

```

Comment:
Can you please improve the description.

Reply:
tried to make a more specified description